src.fork5/Backend/Core/Engine/DataGrid.php

```diff
@@ -283,9 +283,9 @@ public function enableSequenceByDragAndDrop(): void
         }
 
         // add a column for the handle, so users have something to hold while dragging
-        $this->addColumn('dragAndDropHandle', null, '<span class="fas fa-bars" aria-hidden="true"></span>');
+        $this->addColumn('drag-and-drop-handle', null, '<span class="fas fa-bars" aria-hidden="true"></span>');
```

Comment:
Hmm, is dat niet verwarrend voor devs?

Reply:
src.fork5 bestanden zou ik zowieso hier niet aanpassen en die allemaal terug draaien